docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
```

Comment:
@colefreeman Remove the trailing backslash from the url values as well. Note how the default value used [here](https://github.com/mage-ai/mage-ai/pull/5166/files#diff-9ea4ed7bb923d91d2c737acc9fea956a9fb38ccdf5ac174b9550b8266c17f0e7R39) or [here](https://github.com/mage-ai/mage-ai/pull/5166/files#diff-9ea4ed7bb923d91d2c737acc9fea956a9fb38ccdf5ac174b9550b8266c17f0e7R43) has no backslash. That syntax could actually matter in some cases.